src/features/dashboard/components/ApiTokenTable/index.tsx

```diff
@@ -32,35 +42,60 @@ const tableColumns: TTokenColumn[] = [
     Cell: ApiLastUsedCell,
   },
   {
-    Header: 'Valid for IP',
-    accessor: 'valid_for_ip',
+    Header: 'Actions',
+    id: 'actions',
+    accessor: (originalRow) => originalRow.token,
+    Cell: ({ row }) => <TokenActionsCell tokenId={row.original.token} flex_end />,
   },
 ];
 
 const ApiTokenTable = (props: HTMLAttributes<HTMLDivElement>) => {
-  const ROW_HEIGHT = 125;
   const { tokens, isLoadingTokens } = useApiToken();
-  const [table_height, setTableHeight] = useState(0);
+  const { deviceType } = useDeviceType();
+  const [tableHeight, setTableHeight] = useState(0);
+
+  const renderTable = () => {
+    return deviceType === 'desktop' ? (
+      <Table data={tokens} columns={tokenTableColumns()} parentClass='api_token_table' />
+    ) : (
+      <ResponsiveTable tokens={tokens} />
+    );
+  };
 
-  useEffect(() => {
-    if (tokens.length > 0) {
-      setTableHeight(ROW_HEIGHT * tokens.length);
-    }
-  }, [tokens]);
+  const ResponsiveDevice = () => {
+    if (deviceType === 'tablet') return `${styles.api_table} ${styles.tablet}`;
+    if (deviceType === 'mobile') return `${styles.api_table} ${styles.mobile}`;
+    return styles.api_table;
+  };
```

Comment:
We can use `clsx` to achieve the conditional styling. You can look into the `DropdownList` index file for reference.